FaceID/faceID.py

```diff
@@ -0,0 +1,361 @@
+
+"""
+*Testing foundation for FaceID functions*
+
+In order to test, make sure that:
+1. Opencv is successfully installed
+2. Run "source ~/.profile && workon cv" before running program
+   --> this puts you in the virtual environment with all necessary tools
+3. Make sure that the USB webcam is connected through the VM (until built-in is figured out)
+3. Ready! 
+"""
+
+# LIBRARIES
+import cv2
+import os
+import sys
+import numpy as np
+from PIL import Image
+
+
+# GLOBAL VARIABLES
+photoRegister = [] # Stores photo names for each person as a 2D list
+frameCount = 25 # the amount of frames to take and maintain for an employee
+
+# loads face detection model
+face_detector = cv2.CascadeClassifier('Cascades/haarcascade_frontalface_default.xml')
+
+# Reads the contents of the photoNames.txt file
+def readPhotoRegister():
+    
+    global photoRegister # takes updated photoRegister from global variable
+
+    photoNames = open("photoNames.txt", "r") # opens txt file
+    photoRegister = photoNames.readlines() # reads lines 
+    
+    # formats read lines into 2D list
+    for i in range(len(photoRegister)):
+        person = photoRegister[i]
+        person = person.strip().split(",")
+        #print(person)
+        photoRegister[i] = person
+
+    photoNames.close() # closes txt file
+
+    return(photoRegister) # returns updated 2D list
+ 
+
+# Writes new content to the photoNames.txt file
+# This is called whenever an image is written to the dataset folder
+def writePhotoRegister():
+
+    global photoRegister # takes updated photoRegister from global variable
+    photoNames = open("photoNames.txt", "w") # opens txt file
+
+    photoNames.truncate() # delete contents first
+
+    # writes contents to txt file line by line
+    copy = []
+    for person in photoRegister:
+        line = ",".join(person)
+        copy.append(line)
+        photoNames.write(line+"\n")
+
+    photoNames.close() # closes txt file
+
+
+def displayEmployees():
+    photoRegister = readPhotoRegister()
+    if len(photoRegister)>0:
+        print("\nEmployee ID List:")
+        for person in photoRegister:
+            extractEmpID = int(person[0].split("_")[0]) # gets Employee ID
+            print(extractEmpID)
+        print("\n")
+    else: 
+        print("\nNo Registered Employees!\n")
+    return(len(photoRegister))
+
+# Registers a new employee
+def createEmployee():
+
+    # 1. get employee ID
+    # 2. take frames 
+    # 3. append person to photoRegister --> [employeeID] + [photo#]
+    # 4. add to dataset/ folder & send to database
+    # 3 & 4 are both actions that append the new user to the system
+
+    global photoRegister # brings 2D list from global scope
+    global frameCount
+    photoNum = 0 # an index inside of the 'person[]' list
+
+    # Manages input for new employee ID
+    flag = False
+    while flag==False:
+        empID = input("Enter your EMPLOYEE ID:  ") # get employee ID
+        if len(photoRegister)==0: 
+            flag=True
```

Comment:
We should be consistent about spacing between operators (`=`, `==`, `>` ...). I think preferred would be spaces on both sides